snaps_k8s/provision/ansible_configuration.py

```diff
@@ -758,6 +778,13 @@ def __install_kubectl(k8s_conf):
 
     host_name, ip = config_utils.get_first_master_host(k8s_conf)
     ha_enabled = len(lb_ips) > 0
```

Comment:
Please get the HA information from config_utils

Reply:
Hi Steve,

value of  ha_enabled is being fetched from ib_ip which is being extracted from config_utils file.
code snippet-
 lb_ip = "127.0.0.1"
    lb_ips = config_utils.get_ha_lb_ips(k8s_conf)
    if len(lb_ips) > 0:
        lb_ip = lb_ips[0]

    logger.info("Load balancer ip %s", lb_ip)

    host_name, ip = config_utils.get_first_master_host(k8s_conf)
    ha_enabled = len(lb_ips) > 0

Please let me know if you have any input.